src/nat/middleware/defense_middleware.py

```
@@ -0,0 +1,379 @@
+                "is_basemodel": is_basemodel,
+                "original_type": original_type
+            }
+
+            logger.debug(
+                "Extracted field '%s' from value: %s -> %s",
+                self.config.target_field,
+                value,
+                extracted_value
+            )
+
+            return extracted_value, field_info
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)